R/symptome_compartment_class.R

```diff
@@ -0,0 +1,226 @@
+#' An SEIR model for compartments of different symptoms
+#'
+#' Note:
+#' 1) Sum of poulation equals to 1 at any time point.
+#' An error will be reported the sum of initial population is not equal to 1.
+#' 2) At this stage, we only consider disease-caused death cases
```

Comment:
I'm not sure I understand this comment? Do you mean that, at the moment, there are no natural deaths (only those from infected individuals)?

Reply:
Yes that's true - the comment refers to we do not consider natural deaths right now. Sorry for the confusion.